### DISCUSSION TITLE
Vive Focus 3 support

### DISCUSSION THREAD

**Discussion by ZeeNastalski** [alvr-org/ALVR#794] [General]
HTC released their new standalone Vive Focus 3 headset powered by Qualcomm XR2 chip (same as Quest 2). 
The question is, if it is feasible to add support for Focus 3 in ALVR?
Is there anything in ALVR that ties the software to the Oculus platform that would make it difficult to add support to this new headset?

The Focus 3 is aimed at enterprise use cases. It comes with its own VR streaming solution called Vive Business Streaming. It has excellent positional tracking capabilities (better than quest 2 in my testing) and is an excellent choice for location based entertainment because of swappable battery. 

Zee

**Comment by zmerp** [comment]
We are working on a OpenXR version of the client and the Vive Focus 3 will probably be supported.

**Comment by niiln** [comment]
Any progress in Focus 3 support? Fork already support OpenXR, but no version for Focus 3.

**Reply by zmerp** [reply]
The Vive Focus 3 OpenXR support is in closed beta, and even when it comes out it only has support for OpenGL, while @korejan fork only has Vulkan support.

**Reply by niiln** [reply]
Can I join to beta testers group of this version? Official vive streaming app is trash.

**Reply by zmerp** [reply]
No, the beta program is restricted to companies. I got access because I worked with our sponsor Varonia. Unfortunately there is nothing I can share publicly at the moment.

**Reply by niiln** [reply]
Any date for releasing this beta in public?

**Reply by zmerp** [reply]
No, there is no ETA.

**Reply by Pandaily591** [reply]
Focus 3 OpenXR is now in public beta

**Reply by niiln** [reply]
Any new information about releasing alvr to focus 3?

**Reply by zmerp** [reply]
I'm working on some refactorings in the ALVR client that will allow to easily add support to other VR runtimes under OpenGL. I don't have an ETA but that will be soon (probably within a month).

**Reply by niiln** [reply]
Does this mean client support f3 or something else?

**Reply by zmerp** [reply]
Personally I will work on a Pico Neo version for a company but other members of the community can help porting to the Focus 3.
